browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEventPanel/VendorEventPanel.tsx

```diff
@@ -6,6 +6,7 @@ import { Column, ResourceTable } from "@requestly-ui/resource-table";
 import { ObjectInspector } from "@devtools-ds/object-inspector";
 import { CollapseProps } from "antd/lib";
 import { capitalize } from "lodash";
+import { VendorEvent } from "@requestly/analytics-vendors/types";
```

Comment:
We should export the types from `@requestly/analytics-vendors` index.js
as this wont work when npm package is published.

```js
// analytics-vendors index.ts
export * from types;
```

and then here 

```suggestion
import { VendorEvent } from "@requestly/analytics-vendors";
```

For reference: check the structure of requestly-web-sdk's root file

Reply:
Thanks for pointing out!